### ISSUE TITLE
[0.3] Page skipped? 

### ISSUE THREAD

**Issue by bdauria** [futurepress/epub.js#657]
I've noticed a strange issue while testing out epub.js 0.3 (and does not occur on 0.2): There seem to be a page skip on the last page of some paginated content. The following gif illustrates the problem: 

![last-page-skipped](https://user-images.githubusercontent.com/8367233/30661674-0934a924-9e45-11e7-80b1-1def59365c50.gif)

As you can can see, the last part of the chapter is actually skipped, and is visible only after navigating back from the last page. 

The code I'm using is pretty straightforward:

```
  this.book = new Epub('path/to/the/book/, {});
  const rendition = this.book.renderTo('area', {
	method: 'paginate',
	width: 300,
	height: 500,
	styles: '1.60em'
  });
 rendition.display();
```

the area itself is nothing else than a simple div. 

This doesn't seem to be an isolated issue, since it happens for all the epubs that I've tested. Occurs on safari as well as chrome also. 

Any idea where that might be coming from ?

Thanks!

**Comment by respinos** [comment]
It seemed in our experiments that depending on the reader dimensions, if the chapter's layout led to an odd number of pages, that last page would be skipped in normal navigation.

We added this bit in `managers/views/iframe.js` during the calculating of the `horizontal` expand:

```
			// add an extra page if this is odd
			if ( ( width / this.layout.pageWidth ) % 2 > 0 ) {
				width += this.layout.pageWidth;
			}
```

which solved some of the problem. There's commented out code in v.03 (or was!) that addresses the same issue (but I can't recall if we tried that.)

**Comment by bdauria** [comment]
I've tried using this snippet in the iframe.js, it doesn't seem to solve the problem on my side.

**Comment by fchasen** [comment]
Fixed in `epubjs@0.3.52` t
